zenodo/modules/pages/config.py

```diff
@@ -26,10 +26,13 @@
 
 from __future__ import absolute_import, print_function
 
-#: Description minimum length
+#: Maximum size of attachment in contact form.
+PAGES_ATTACHMENT_SIZE = 10
```

Comment:
We have a convention to express file sizes like this:
https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/config.py#L41

``PAGES_ATTACHMENT_MAX_SIZE = 1000 * 1000 * 10  # 10 MB``